locale/en/blog/uncategorized/10-lts-to-12-lts.md

```diff
@@ -0,0 +1,4333 @@
+---
+date: 2019-10-20T12:00:00.000Z
+category: Uncategorized
+title: The Difference Between Node.js 10 LTS and Node.js 12 LTS
+slug: the-difference-between-node-js-10-lts-and-node-js-12-lts
+layout: blog-post.hbs
+author: Tierney Cyren
+---
+
+With the transition of Node.js 12 from LTS into Current, a suite of new features have been added to Node.js!
```

Comment:
Node.js 12 is not transitioning from LTS into Current, but the other way, right?

Reply:
The new features aren't really new in Node.js, are they? They're new in LTS, perhaps, but they were already there in 12.x and now the _very_ new features are in 13.x.

I'm not sure this sentence adds a whole lot, but the post does need *some* kind of introductory sentence. Maybe something more like this?:

> We are pleased to announce that Node.js 12 is now the Active LTS version of Node.js.